examples/deep_autoencoder.py

```diff
@@ -0,0 +1,12 @@
+""" A demonstration of how to use the Deep Autoencoder. Using the small iris dataset.
+"""
+
+
+
+from sklearn import datasets
+import skflow
+iris = datasets.load_iris()
+x = iris.data
+
+result = skflow.StackedAutoEncoder(x, dims=[5, 4, 3], noise='gaussian', epoch=1000).encode()
```

Comment:
API can look:

```
result = skflow.StackedAutoEncoder(dims=[5, 4, 3], noise='gaussian', epoch=1000).fit_transform(x)
```

This way you can also create it, fit it on one dataset and encode other datasets.